build/downgrade-rector.sh

```diff
@@ -36,3 +36,8 @@ for directory in $directories; do
     # --working-dir is needed, so "SKIP" parameter is applied in absolute path of nested directory
     php -d memory_limit=-1 bin/rector process $directory --config build/config/config-downgrade.php --working-dir $BUILD_DIRECTORY --ansi
 done
+
+
+# CONFIRMED: give time to print all the files, before the next process takes over newly printed content
+# avoids bugs like these half of files done, next half waiting https://github.com/rectorphp/rector-src/runs/2565478682
+sleep 20
```

Comment:
@samsonasik Hey, this was one of reasons why some code was not downgraded corretly.
It printed too slow and next process took over the old code :) it might help you with PHP 7.0 build

Reply:
@TomasVotruba thank you 👍